qa/vagrant-helpers.rb

```diff
@@ -34,10 +50,17 @@ def self.parse(lines)
 
     def self.execute(cmd)
       Open3.popen3(cmd) do |stdin, stdout, stderr, wait_thr|
-        { :stdout => stdout.read.chomp, :stderr => stderr.read.chomp,
-          :exit_status => wait_thr.value.exitstatus }
+        ExecuteResponse.new(stdin, stdout.read.chomp, stderr.read.chomp, wait_thr.value.exitstatus)
```

Comment:
it is nice to have a bit more structure, and the added sugar of methods like `success?` I also feel like this structure will lend itself to possible future improvements where we want to wrap async executions (background processes).